data-model/SDP-sample-records/textual-artifact_sample1.json

```diff
@@ -0,0 +1,65 @@
+{
+  "id": "ark:/21198/adf20c1f",
+  "type": "Textual Artifact",
+  "label": "Sinai Syriac 2A, ff. 1-152v, Gospels",
```

Comment:
For textual artifacts the label can just be the work. Simple, human-readable rather than anything authoratative